### ISSUE TITLE
Shinken complains about missing library

### ISSUE THREAD

**Issue by mohierf** [naparuba/shinken#1715]
After a fresh install, Shinken always complain about a missing Python library

Shinken CLI:
```
$ shinken inventory
[1441641004] ERROR: [Shinken] Cannot find python lib crypto: export to kernel.shinken.io isnot nsca
```

In Shinken.log: 
```
[1441640767] ERROR: [Shinken] Cannot find python lib crypto: export to kernel.shinken.io isnot available
```

As I do not want to export to kernel.shinken.io, I would appreciate not to have those ERROR logs !

**Comment by tson0n0** [comment]
I have the same probleme.

**Comment by olivierHa** [comment]
It should be fix in version 2.4.1. Could you check your shinken version ?

2015-09-24 14:26 GMT+02:00 tson0n0 <notifications@github.com>:
